pipeline/inputs/windows-exporter-metrics.md

```diff
@@ -90,9 +95,89 @@ You can test the expose of the metrics by using _curl:_
+
+To reduce the amount of this metrics, users have to use WHERE clause.
+This syntax is defined in [the WMI Query Language(WQL)](https://learn.microsoft.com/en-us/windows/win32/wmisdk/wql-sql-for-wmi).
+
+Here is how these parameters should work:
+
+#### we.service.where
+
+`we.service.where` is just handled as normal where clause.
+When a user specify the parameter as follows:
+
+```
+we.service.where Status!='OK'
+```
+
+The WMI query should be called as:
+```
+SELECT * FROM Win32_Service WHERE Status!='OK'
+```
+
+Then, WMI mechanism should return the information which has not OK status in this example.
+
+### we.service.include
+
+`we.service.include` is interpreted into where clause.
+If the multiple key-value pairs are specified, the values will be concatenated with `OR`.
+Also, the values contain `%` character, `LIKE` operator should be used in the transrated clause instead of `=` operator.
+When a user specify the parameter as follows:
+
+```python
+we.service.include {"Name":"docker","Name":"%Svc%", "Name":"%Service"}
+```
+
+The parameter should be interpreted as:
+```
+(Name='docker' OR Name LIKE '%Svc%' OR Name LIKE '%Service')
+```
+
+The WMI query should be called with the translated parameter as:
+
+```
+SELECT * FROM Win32_Service WHERE (Name='docker' OR Name LIKE '%Svc%' OR Name LIKE '%Service')
+```
+
+### we.service.exclude
+
+`we.service.where` is interpreted into where clause.
+If the multiple key-value pairs are specified, the values will be concatenated with `AND`.
+Also, the values contain `%` character, `LIKE` operator should be used in the translated clause instead of `!=` operator.
+When a user specify the parameter as follows:
+
+```python
+we.service.exclude {"Name":"UdkUserSvc%","Name":"webthreatdefusersvc%","Name":"XboxNetApiSvc"}
+```
+
+The parameter should be interpreted as:
+```
+(NOT Name LIKE 'UdkUserSvc%' AND NOT Name LIKE 'webthreatdefusersvc%' AND Name!='XboxNetApiSvc')
+```
+
+The WMI query should be called with the translated parameter as:
+
+```
+SELECT * FROM Win32_Service WHERE (NOT Name LIKE 'UdkUserSvc%' AND NOT Name LIKE 'webthreatdefusersvc%' AND Name!='XboxNetApiSvc')
+```
+
+### advanced usages
+
+`we.service.where`, `we.service.include`, and `we.service.exclude` can be used in the same time.
+However, these parameters are prioritized as follows:
+
+1. `we.service.include` translated and applied into the where clause in the service collector
+1. `we.service.exclude` translated and applied into the where clause in the service collector
+    1. If the `we.service.include` is applied, translated `we.service.include` and `we.service.exclude` conditions are concatenated with `AND`.
+1. `we.service.where` is just handled as-is into the where clause in the service collector .
+    1. If either of the above parameters is applied, the clause will be applied with `AND (` _the value of `we.service.where`_ `)`.
+
```

Comment:
Maybe a simple example here?

Reply:
Sure. Let's add example for this section.

Reply:
I added an example for this case: https://github.com/fluent/fluent-bit-docs/pull/1140/commits/efab53fd419f4b0af7847cbda13402ec37c3324a